src/services/todo_service.py

```diff
@@ -0,0 +1,152 @@
+import sqlite3 as sql
+from pathlib import Path
+from flask import jsonify, make_response
+from datetime import datetime
+
+
+def user_id_exist(data_file, user_id):
+    connection = sql.connect(data_file)
+    cursor = connection.cursor()
+    statement = "SELECT user_id FROM USERS WHERE user_id = ?;"
+    cursor.execute(statement, (user_id,))
+    sql_file = cursor.fetchall()
+    return bool(sql_file)
+
+
+def task_id_exist(data_file, task_id):
+    connection = sql.connect(data_file)
+    cursor = connection.cursor()
+    statement = " SELECT task_id FROM TASKS WHERE task_id = ?;"
+    cursor.execute(statement, (task_id,))
+    sql_file = cursor.fetchall()
+    if len(sql_file) == 0:
+        return False
+    return True
+
+
+def get_all_todo():
+    data_file = Path(Path.cwd(), "data", "data_sql.db")
+    connection = sql.connect(data_file)
+    cursor = connection.cursor()
+
+    statement = "SELECT * FROM TASKS"
+    cursor.execute(statement)
+    sql_data = cursor.fetchall()
+    tasks = [{"task_id": task[0], "user_id": task[1], "title": task[2], "description": task[3], "status": task[4],
```

Comment:
I would export creating dictionary to separte function, so you could use it in other places as well

[create_user_dict(task[0], task[1], task[2], task[3], task[4], task[5], task[6]) for task in sql_data]

Reply:
i created tasks_response_builder for one task because i wanted use it in get single task, in get all task and get all task from user i use it with list comprehension